src/Toolkit/Toolkit/UI/Controls/FeatureForm/TextFormInputView.Maui.cs

```diff
@@ -63,13 +66,18 @@ private static object BuildDefaultTemplate()
             Border errorBorder = new Border() { StrokeThickness = 1, Stroke = new SolidColorBrush(Colors.Red), IsVisible = false };
             Grid.SetColumnSpan(errorBorder, 2);
             root.Add(errorBorder);
+            ImageButton barcodeButton = new ImageButton() { IsVisible = false, BorderWidth = 0, Source = new FontImageSource() { FontFamily = "calcite-ui-icons-24", Glyph = "\uE22F", Color = Colors.Black } };
```

Comment:
The icon is rather misaligned.  And in dark mode, it's almost invisible (black-on-dark-gray):

![2024-07-15_013317](https://github.com/user-attachments/assets/4312eef9-4e15-4ea5-a807-19f4161d8726)